frontend/src/views/MovieSessionDetailsScreen.vue

```diff
@@ -95,7 +95,7 @@ export default {
         }
       };
 
-      await this.axios.post(`${import.meta.env.VITE_API_URL}/api/cinema/orders`, { tickets },
+      await this.axios.post(`${import.meta.env.VITE_API_URL}/api/cinema/orders/`, { tickets },
```

Comment:
Are you sure you fixed all endpoints in this and other files?
Also don't see screenshot of all responses from developer tools